docs/en/solutions/How_to_Deploy_nmstate.md

```
@@ -0,0 +1,261 @@
+node1          2d19h
+node2           2d19h
+node3           2d19h
+```
+
+#### Configure Underlay Bond Network Card
+
+**Prerequisites:** The underlay physical network cards must not be mounted on any OVS. You can check with `ip l show {network_card} |grep ovs`. If the output is empty, it means it's not mounted.
+
+Configuration YAML as follows:
+
+**Bond Network Configuration**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)